README.md

```diff
@@ -116,6 +114,30 @@ ReactDOM.render((
 ), document.getElementById('slider-root'));
 ```
 
+## Using `react-with-styles`
+
+Rather than styling the component with css, we use `react-with-styles`, an interface compatible with
+React Native, Aphrodite, JSS, and more to come. Full instructions for `react-with-styles` can be
+found [here](https://github.com/airbnb/react-with-styles), but the base implementation requires:
```

Comment:
Linking "here" or "click here" should generally be avoided in favor of more descriptive link text. What do you think about rephrasing this as

```md
[Full instructions for `react-with-styles` are available](https://github.com/airbnb/react-with-styles), but the base ...
```